models.js

```diff
@@ -47,4 +47,23 @@ function getArticleAndSort(sort_by = 'created_at', order = 'desc'){
     })
 }
 
-module.exports={getTopics, getAllEndPoints, getArtcileById, getArticleAndSort}
\ No newline at end of file
+function getCommentsForArticle(article_id, sort_by = 'created_at', order = 'desc'){
+    const validSortBys = ['created_at']
+
+    if (!validSortBys.includes(sort_by)) {
+        return Promise.reject({ status: 400, message: 'invalid query value'})
+    }
+
+    let sqlString = `SELECT * FROM comments WHERE article_id = $1 `;
+    sqlString += `ORDER BY ${sort_by} ${order.toUpperCase()}`;
```

Comment:
We are exposing ourselves to SQL injection here with the order as we haven't greenlit certain values above. Again we didn't need to include queries yet to satisfy this ticket, but if we are going to do it we need to be careful to avoid SQL injection and ensure the queries are fully tested.